packages/ubuntu_desktop_installer/lib/pages/who_are_you/who_are_you_page.dart

```diff
@@ -0,0 +1,368 @@
+                                screenFactor,
+                            child: Form(
+                              key: _confirmPasswordFormKey,
+                              child: TextFormField(
+                                autovalidateMode:
+                                    AutovalidateMode.onUserInteraction,
+                                controller: _confirmPasswordController,
+                                validator: (val) {
+                                  return MatchValidator(
+                                          errorText: lang
+                                              .whoAreYouPageConfirmPasswordValidatorErrorText)
+                                      .validateMatch(
+                                          val!, _passwordController.text);
+                                },
+                                obscureText: true,
+                                decoration: InputDecoration(
+                                    border: OutlineInputBorder(),
+                                    labelText:
+                                        lang.whoAreYouPageConfirmPasswordLabel),
+                              ),
+                            ),
+                          ),
+                          ValueListenableBuilder<TextEditingValue>(
+                              valueListenable: _confirmPasswordController,
+                              builder: (context, value, child) {
+                                return Padding(
+                                  padding: const EdgeInsets.only(
+                                      left: checkMarksLeftPadding),
+                                  child:
+                                      value.text != _passwordController.text ||
+                                              !MultiValidator([
+                                                RequiredValidator(
+                                                    errorText: lang
+                                                        .whoAreYouPagePasswordRequiredValidatorErrorText),
+                                                MinLengthValidator(2,
+                                                    errorText: lang
+                                                        .whoAreYouPagePasswordMinLengthValidatorErrorText),
+                                              ]).isValid(value.text)
+                                          ? Text('')
+                                          : successIcon,
+                                );
+                              })
+                        ],
+                      ),
+                    ),
+                    Row(
+                      children: [
+                        Radio<LoginStrategy>(
+                            value: LoginStrategy.autoLogin,
+                            groupValue: _whoAreYouModel.loginStrategy,
+                            onChanged: (_) => setState(() {
```

Comment:
please remove all `setState()` calls from the view. accessing the model in "listen" mode in the `build` method triggers rebuilds whenever the model notifies state changes